gallery/visualtests/nxCheckbox.js

```diff
@@ -19,7 +19,7 @@ describe('NxCheckbox', function() {
     it('has a light grey border and white background by default', simpleTest(selector));
     it('has a black border when hovered', hoverTest(selector));
 
-    it('has a blue background and white checkmark when clicked', async function() {
+    it('has a blue background and white checkmark and a light blue outer border when clicked', async function() {
```

Comment:
The test below is for when it is clicked and focused, so this was supposed to be the test for when it is clicked and _not_ focused. Maybe it was failing to be that - all the more indication that the old focus style was too hard to see.  But my point is that I don't think this test should be expecting the focus style, and it should be modified so that the element is not in fact focused (or hovered) when the screenshot is taken

Reply:
> The test below is for when it is clicked and focused, so this was supposed to be the test for when it is clicked and _not_ focused. Maybe it was failing to be that - all the more indication that the old focus style was too hard to see. But my point is that I don't think this test should be expecting the focus style, and it should be modified so that the element is not in fact focused (or hovered) when the screenshot is taken

I accidentally added the "focused" test when the checkbox/radio is clicked and _not_ focused. This has been fixed.
982993dfc75068be14ca66b90b7028342c5923e4

Reply:
> I accidentally added the "focused" test when the checkbox/radio is clicked and not focused. This has been fixed.
982993d

Does the focus border show up in the applitools baseline for this test? I know the test doesn't explicitly focus the input, but generally things get focus when you click on them, which is why I'm thinking this test, as currently written, would actually pick up the focus state too.

Reply:
> > I accidentally added the "focused" test when the checkbox/radio is clicked and not focused. This has been fixed.
> > [982993d](https://github.com/sonatype/sonatype-react-shared-components/commit/982993dfc75068be14ca66b90b7028342c5923e4)
> 
> Does the focus border show up in the applitools baseline for this test? I know the test doesn't explicitly focus the input, but generally things get focus when you click on them, which is why I'm thinking this test, as currently written, would actually pick up the focus state too.

The focus border previously (while clicked) was so subtle that it's incredibly difficult to see if there's even a focus border present. It's really difficult to tell whether the baseline picks up the focus state or not, both for the radios and checkboxes.

Checkbox clicked baseline:
![image](https://user-images.githubusercontent.com/7110862/124013146-f6ea8180-d9af-11eb-89ed-059a156c83b8.png)

Radio clicked baseline:
![image](https://user-images.githubusercontent.com/7110862/124013422-46c94880-d9b0-11eb-94cc-818673ec2198.png)

Reply:
> The focus border previously (while clicked) was so subtle that it's incredibly difficult to see if there's even a focus border present. It's really difficult to tell whether the baseline picks up the focus state or not, both for the radios and checkboxes.

In the past, sure.  But that doesn't matter now thanks to this branch.  Are the new focus borders visible in the screenshot for this test, in this branch?

Reply:
> Yes, the focus borders are visible for this test (Default NxCheckbox has a blue background and white checkmark when clicked)

Which makes it effectively identical to the next test below, which isn't what we want.  So this test needs to be modified to blur the element after clicking on it

Reply:
> > Yes, the focus borders are visible for this test (Default NxCheckbox has a blue background and white checkmark when clicked)
> 
> Which makes it effectively identical to the next test below, which isn't what we want. So this test needs to be modified to blur the element after clicking on it

Added blur after click to both checkboxes and radiobuttons.
7a75d5873a9038d9f66c7cd22b1bfae945ae03b1